jetty-start/src/test/java/org/eclipse/jetty/start/fileinits/MavenMetadataTest.java

```diff
@@ -0,0 +1,107 @@
+
+import org.eclipse.jetty.toolchain.test.MavenTestingUtils;
+import org.junit.jupiter.api.Test;
+import org.xml.sax.SAXException;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class MavenMetadataTest
+{
+    private static final TimeZone UTC = TimeZone.getTimeZone("UTC");
+
+    @Test
+    public void testParseExample() throws ParserConfigurationException, SAXException, IOException
+    {
+        Path example = MavenTestingUtils.getTestResourcePathFile("example-maven-metadata.xml");
+        MavenMetadata mavenMetadata = new MavenMetadata(example);
+
+        assertThat("Metadata.groupId", mavenMetadata.getGroupId(), is("org.eclipse.jetty"));
+        assertThat("Metadata.artifactId", mavenMetadata.getArtifactId(), is("jetty-rewrite"));
+        assertThat("Metadata.version", mavenMetadata.getVersion(), is("10.0.0-SNAPSHOT"));
+
+        assertThat("Metadata.versioning.lastUpdated", mavenMetadata.getLastUpdated(), is("20200918022411"));
+
+        assertThat("Metadata.versioning.snapshot.timestamp", mavenMetadata.getSnapshotTimestamp(), is("20200918.022411"));
+        assertThat("Metadata.versioning.snapshot.buildNumber", mavenMetadata.getSnapshotBuildNumber(), is("580"));
+
+        assertThat("Metadata.snapshots.size", mavenMetadata.getSnapshots().size(), is(4));
+
+        assertThat("Metadata.snapshot(null, 'jar').value",
+            mavenMetadata.getSnapshot(null, "jar").getValue(),
+            is("10.0.0-20200918.022411-580"));
+    }
+
+    @Test
+    public void testIsExpiredTimestampVeryOld()
+    {
+        String timestamp = "20190822223344";
+        assertTrue(MavenMetadata.isExpiredTimestamp(timestamp), "Timestamp should be stale: " + timestamp);
+    }
+
+    @Test
+    public void testIsExpiredTimestampNextWeek()
+    {
+        LocalDateTime nextWeek = LocalDateTime.now().plusWeeks(1);
+        String timestamp = getTimestampFormatter().format(nextWeek);
+        assertFalse(MavenMetadata.isExpiredTimestamp(timestamp), "Timestamp should NOT be stale: " + timestamp);
+    }
+
+    @Test
+    public void testIsExpiredTimestampNow()
+    {
+        LocalDateTime now = LocalDateTime.now();
+        String timestamp = getTimestampFormatter().format(now);
+        assertFalse(MavenMetadata.isExpiredTimestamp(timestamp), "Timestamp should NOT be stale: " + timestamp);
+    }
+
+    @Test
+    public void testIsExpiredTimestampYesterday()
+    {
+        LocalDateTime yesterday = LocalDateTime.now().minusDays(1);
+        String timestamp = getTimestampFormatter().format(yesterday);
+        assertTrue(MavenMetadata.isExpiredTimestamp(timestamp), "Timestamp should be stale: " + timestamp);
+    }
+
+    @Test
+    public void testIsExpiredTimestamp12HoursOld()
+    {
+        LocalDateTime oldish = LocalDateTime.now().minusHours(12);
```

Comment:
what about something such `now(ZoneId.of("UTC"))` as `getTimestampFormatter` use `UTC`?

Reply:
That might work.

Reply:
I opted to just remove this specific test, it's not reliable as written, and is too sensitive to the time of day that surefire user executes the test on.